### DISCUSSION TITLE
Can't connect to Dahua NVR

### DISCUSSION THREAD

**Discussion by andlinux007** [bluecherrydvr/unity#253] [Support]
I downloaded the AppImage and at the beginning it asks the username, password and ports 
but I guess I tried all the ports I can think of, I also did a portscan on my nvr and that showed 80 and 443 so I entered those but no luck, it keeps complaining about the ports. Does anybody know what I'm doing wrong?
![1](https://github.com/user-attachments/assets/65884223-f8d9-4303-a416-6cb36226269e)

**Comment by curtishall** [comment]
The Bluecherry client was originally designed to work with our server software. However recently we added support for connecting to external RTSP streams that are not from our server.

You can hit skip then follow the steps to add a custom rtsp path, you would need to know the rtsp for try Dahua NVR.
